pom.xml

```diff
@@ -393,6 +397,13 @@
   <!-- ==================================================================== -->
   <build>
     <resources>
+      <resource>
+        <directory>${project.basedir}/src/main/resources/META-INF</directory>
+        <targetPath>META-INF</targetPath>
+        <includes>
+          <include>**/*.json</include>
```

Comment:
I would prefer a more precise definition here - I'm guessing that would be `native-image/joda-time/joda-time/resource-config.json`